bin/opteadm/src/bin/opteadm.rs

```diff
@@ -478,6 +474,22 @@ fn main() -> anyhow::Result<()> {
             hdl.set_v2p(&req)?;
         }
 
+        Command::SetV2B { prefix, tunnel_endpoint } => {
+            let hdl = opteadm::OpteAdm::open(OpteAdm::XDE_CTL)?;
+            let tep = TunnelEndpoint {
+                ip: tunnel_endpoint,
+                vni: Vni::new(99u32).unwrap(),
```

Comment:
Can we have `BOUNDARY_SERVICES_VNI` as a shared definition somewhere?

Reply:
Done